src/domain/event/eventHero/EventHero.tsx

```diff
@@ -79,6 +80,9 @@ const EventHero: React.FC<Props> = ({ event, eventType }) => {
     }
   }, [imageUrl]);
 
+  const superEventStartTime = superEvent ? superEvent.startTime : startTime;
```

Comment:
`superEventStartTime` might be a misleading name as it is not necessary super event start time. Perhaps rename `startTime` to `eventStartTime` and use:
 `const startTime = superEvent ? superEvent.startTime : eventStartTime;`

Reply:
ok

Reply:
you can rename it:
```
{
...
startTime: eventStartTime,
endTIme: eventEndTime,
...
} = useEventQuery(...);
```

Reply:
i know that i can:) lets not overcomplicate

Reply:
fixed